### PR TITLE
Add table of contents to dev-db readme file

### PR DESCRIPTION
**Pull request by OskarSchamardin**
Add table of contents to readme file located in the dev-db folder. The ToC helps navigate to the desired parts of the file.

Also fix minor formatting inconsistency at the bottom of the file.

### CODE REVIEW

**Review by Orzelius — APPROVED** [review]
Good idea

**Comment by Orzelius** [comment]
@OskarSchamardin If you're want to get stuff merged master you can request review from more people. Then also other's will get better visibility. I recommend to tag your teammate @lupus117 on everything you do.

File: be/db/dev-db/README.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ as:
...
+
+- [Stopping the development database](#stopping-the-development-database)
+
+- [Adding example data and tables](#adding-example-data-and-tables)
+
+- [Troubleshooting](#troubleshooting)
+
 ## Accessing the database
 
 Access the database using adminer at: 'localhost:8080'. Make sure to select
```
Comment (Kasparsu) [inline be/db/dev-db/README.md:25]: maybe add proper link so it would be clickable.

**Review by Kasparsu — CHANGES_REQUESTED** [review]

File: be/db/dev-db/README.md (hunk 1 above)
Comment (Orzelius) [reply be/db/dev-db/README.md:25]: It is clickable. try here -> https://github.com/TPT-Loane/TPT-Loane/tree/db-documentation/be/db/dev-db

**Review by lupus117 — DISMISSED** [review]

File: be/db/dev-db/README.md (hunk 1 above)
Comment (Kasparsu) [reply be/db/dev-db/README.md:25]: 'localhost:8080' is not clickable! it doesn't even have proper protocol.

Comment (Orzelius) [reply be/db/dev-db/README.md:25]: aaa, I thought you meant the toc links.
In this case, might as well add some parameters to the url to automatically fill in the user and db parts.
`http://localhost:8080/?server=db&username=postgres&db=postgres`

**Review by Kasparsu — CHANGES_REQUESTED** [review]
https://github.com/TPT-Loane/TPT-Loane/pull/34/files#r617828362

File: be/db/dev-db/README.md (hunk 1 above)
Comment (Kasparsu) [reply be/db/dev-db/README.md:25]: @OskarSchamardin fix plz

**Review by Orzelius — APPROVED** [review]

**Review by lupus117 — APPROVED** [review]

**Review by Kasparsu — APPROVED** [review]

**Pull request merged**